lecli/team_api.py

```diff
@@ -92,13 +93,13 @@ def create_team(name):
         response = requests.post(_url(), json=params, headers=headers)
         if response_utils.response_error(response):
             print 'Creating team failed, status code: %d' % response.status_code
```

Comment:
if we are adjusting things, this error message probably should also go to stderr